DQM/SiPixelPhase1Summary/src/SiPixelPhase1Summary.cc

```diff
@@ -311,6 +311,27 @@ void SiPixelPhase1Summary::fillTrendPlots(DQMStore::IBooker & iBooker, DQMStore:
     }
   }
 
+  if (!runOnEndLumi_) return; // The following only occurs in the online
+  //Reset some MEs every 10LS here
+  for (auto it : {1,2,3,4}) { //PXBarrel
+    histNameStream.str("");
+    histNameStream << "PixelPhase1/Phase1_MechanicalView/PXBarrel/clusterposition_zphi_PXLayer_" << it;
```

Comment:
hi @leggat - please just use
histName = "PixelPhase1/Phase1_MechanicalView/PXBarrel/clusterposition_zphi_PXLayer" + std::to_string(it);